src/scripts/commands/indicators.ts

```diff
@@ -0,0 +1,121 @@
+import _ from "lodash";
+import { command, option, optional, subcommands } from "cmd-ts";
+import { getApiUrlOption, getD2Api, IdsSeparatedByCommas, FilePath, periodYears } from "scripts/common";
+import log from "utils/log";
+import { IndicatorsD2Repository } from "data/IndicatorsD2Repository";
+import { GetIndicatorNumDenIDsUseCase } from "domain/usecases/GetIndicatorNumDenIDsUseCase";
+import { DataSetsD2Repository } from "data/DataSetsD2Repository";
+import { DataValuesD2Repository } from "data/DataValuesD2Repository";
+import { GetIndicatorsDataElementsValuesReportUseCase } from "domain/usecases/GetIndicatorsDataElementsValuesReportUseCase";
+
+export function getCommand() {
+    const getRefMetadata = command({
+        name: "get-ref-ids",
+        description: "Get the IDs referenced by the Indicators numerators and denominators.",
+        args: {
+            url: getApiUrlOption({ long: "url" }),
+            indicatorsIDs: option({
+                type: IdsSeparatedByCommas,
+                long: "indicators",
+                short: "i",
+                description: "IND1,IND2,...",
+            }),
+            dataSetFilterList: option({
+                type: optional(IdsSeparatedByCommas),
+                long: "ds-filter",
+                short: "i",
+                description: "DS1,DS2,...",
+            }),
+            path: option({
+                type: optional(FilePath),
+                long: "file",
+                short: "f",
+                description: "CSV output path (file or directory)",
+            }),
```

Comment:
I get `Path doesn't exist`, as `FilePath` validates the existence of the file. I guess it does not need to exist, for this action?

Reply:
It was a bug in the FilePath checker.